projects/canopy/src/lib/card/card.notes.ts

```diff
@@ -30,28 +30,86 @@ or for a Product card:
 
 ~~~html
 <lg-card>
-  <lg-card-header>
-    <lg-card-title headingLevel="3">
-      Card Title
-    </lg-card-title>
-    <lg-card-subtitle>
-      Payroll Reference Number P23456
-    </lg-card-subtitle>
-    <lg-card-principle-data-point>
-      <lg-card-principle-data-point-label>
-        Last payment (after tax and deductions)
-      </lg-card-principle-data-point-label>
-      <lg-card-principle-data-point-value>
-        <span><span class="lg-font-size-3">£</span>230.20</span>
-      </lg-card-principle-data-point-value>
-      <lg-card-principle-data-point-date>
-        as of 01 Jan 2020
-      </lg-card-principle-data-point-date>
-    </lg-card-principle-data-point>
-  </lg-card-header>
+  <lg-card-content>
+    <div lgRow>
+      <div lgCol=12 lgColMd=6>
+        <lg-card-title headingLevel="4">
+          <a href="#">{{title}}</a>.
+        </lg-card-title>
+        <lg-card-subtitle>
+          Payroll Reference Number P23456
+        </lg-card-subtitle>
+      </div>
+      <lg-card-principle-data-point lgCol=12 lgColMd=6>
+        <lg-card-principle-data-point-label>
+          Last payment (after tax and deductions)
+        </lg-card-principle-data-point-label>
+        <lg-card-principle-data-point-value>
+          <span><span class="lg-font-size-3">£</span>230.20</span>
+        </lg-card-principle-data-point-value>
+        <lg-card-principle-data-point-date>
+          as of 01 Jan 2020
+        </lg-card-principle-data-point-date>
+      </lg-card-principle-data-point>
+    </div>
+  </lg-card-content>
 </lg-card>
 ~~~
 
+or for a Form Journey card:
+
+~~~html
+<div lgContainer>
+  <div lgRow>
+    <div lgCol="12" lgColLg="6" lgColLgOffset="3" lgColMd="10" lgColMdOffset="1">
+      <lg-card lgPadding="none">
+        <lg-card-header lgPadding="sm" lgPaddingBottom="xs" lgMarginBottom="lg">
+          <lg-breadcrumb lgMarginBottom="none">
+            <lg-breadcrumb-item>
+              <a href="#">
+                <lg-icon [name]="'chevron-left'"></lg-icon>
+                Back
+              </a>
+            </lg-breadcrumb-item>
+          </lg-breadcrumb>
+        </lg-card-header>
+        <lg-card-content>
+          <div lgContainer>
+            <div lgRow>
+              <div lgCol="12" lgColMd="10" lgColMdOffset="1">
+                <lg-card-title headingLevel="4">
+                {{ title }}
+                </lg-card-title>
+                <lg-separator></lg-separator>
```

Comment:
This component has a `role="separator"` which in my opinion is going to be wrong in most cases as most of the time we'll use it just for styling. To get around that I think we should add a `aria-hidden="true"` attribute to this instance so that screen readers will skip it. This is because in this case the heading doesn't have to be "separated" from the content it refers to.

I'll open a discussion to talk about the separator. For the moment the attribute in this specific case will be fine.

Reply:
Done.